packages/app/src/providers/mock/mock-api.ts

```diff
@@ -75,6 +81,39 @@ export class MockApi extends DataProviderApi {
     return sliceValue(value, dataset, selection);
   }
 
+  public getExportURL<D extends Dataset<ArrayShape>>(
+    dataset: D,
+    selection: string | undefined,
+    value: Value<D>,
+    format: ExportFormat
+  ): ExportURL {
+    if (this._getExportURL) {
+      this._getExportURL(dataset, selection, value, format);
+    }
+
+    if (
```

Comment:
For testing purposes, I'm implementing a very basic client-side CSV export for 1D numeric datasets (`/nD_datasets/oneD`, `/nD_datasets/oneD_linear`, etc.)

Reply:
Great initiative :+1: